product_docs/docs/migration_toolkit/55/07_invoking_mtk/mtk_command_options_in_file/executing_migration_with_txt.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: "Executing a migration with a command options file"
+navTitle: "Executing a migration with the options file"
+deepToC: true
+---
+
+After you have created the options file, execute the migration command by referencing the options file:
+
+```shell
+./runMTK.sh -optionsFile <example.options>
+```
+
+!!!note
+   Provide the full path for `<example.options>` if you are running the MTK command from a different folder than where you have stored the file.
+!!!
+
+You can specify all options in the file, or use a mix of command line and options file to specify the migration parameters.
+
+Here are some examples for specifying all options in the file:
+
+## Example 1
+
+Content of the `<example.options>` text file:
+
+```
+tables comp_schema.emp,comp_schema.dept,finance_schema.acctg
+```
+
+Syntax of the migration command:
+
+```shell
+./runMTK.sh -optionsFile example.options schema_name
+```
+
+Command line equivalent:
+
+```shell
+./runMTK.sh -tables comp_schema.emp,comp_schema.dept,finance_schema.acctg schema_name
+```
+
+## Example 2
```

Comment:
This example doesn't appear to be significantly different than the Example 1 other than different/addition options are specified in the option file.  I would suggest using this example to show specifying some options via the command line and some in a command file.  I would also label the example as such (e.g., "Migrating a specific schema specifying some options via the command line and others via an options file").

For the example, I would suggest moving the `safeMode` and `connRetryCount 7` options out of the option file and onto the command line. Then run the command as follows:
`./runMTK.sh -safeMode -connRetryCount 7 -optionsFile example.options schema_name`

where the command line equivalent would be:
`./runMTK.sh -safeMode -connRetryCount 7-allTables -excludeTables comp_schema.emp,finance_schema.jobhist -funcs finance_schema.add_two_numbers  schema_name`

Reply:
I updated examples. 
- add titles
- second example has some extra comments, it demonstrates advantage of using Options File to separate configuration, from table definiton. It also links to JIRA issues in comments etc... 
- second example has config parameters (-safeMode...) separated on CLI
-